XenModel/FriendlyNames.resx

```diff
@@ -1841,4 +1841,58 @@
   <data name="Label-host.edition-desktop-cloud" xml:space="preserve">
     <value>XenApp/XenDesktop [Citrix] Cloud</value>
   </data>
+  <data name="Label-host.edition-xcp-ng" xml:space="preserve">
+    <value>XCP-NG</value>
+  </data>
+  <data name="Label-performance.CPU0-avg-freq" xml:space="preserve">

```

Comment:
This can be better achieved with a single resource string and regular expressions. See how we use _Label-performance.cpu_ in https://github.com/xenserver/xenadmin/blob/master/XenModel/Utils/Helpers.cs#L1039.

We would like to take this change, but please update it with the suggestion above.

Reply:
Solution with RegEx: https://github.com/xcp-ng/xenadmin/commit/abe3964cb6fd4ab546e0a6d35a24e7ed345d3bcb

Reply:
Thank you @borzel. Would you like to update this pull request with your solution?